docs/xamarin-forms/platform/ios/ipad-page-presentation-style.md

```diff
@@ -49,7 +49,7 @@ In addition, the `GetModalPresentationStyle` method can be used to retrieve the
 
 The result is that the modal presentation style on a [`Page`](xref:Xamarin.Forms.Page) can be set:
 
-[![](page-presentation-style-images/modal-presentation-style-small.png "Modal Presentation Styles on an iPad")](page-presentation-style-images/modal-presentation-style-large.png#lightbox "Modal Presentation Styles on an iPad")
+[![Modal Presentation Styles on an iPad")](page-presentation-style-images/modal-presentation-style-large.png#lightbox "Modal Presentation Styles on an iPad](page-presentation-style-images/modal-presentation-style-small.png)
```

Comment:
uh oh, this breaks in a number of ways 😕 it swaps the image URLs around, and places the `#lightbox` on the wrong one.

I'm up to 338/423 files, but will need to revisit in case I missed any.